modules/python/Makefile.style

```diff
@@ -44,26 +44,74 @@ python/autoflake: bh/venv
 .PHONY: python/isort/check
 ## Check python imports using isort
 python/isort/check:
-	@$(DOCKER) pull alphachai/isort
 	@$(MAKE) python/lint/announce/isort
+	@if [[ -n "$$(jq -r <Pipfile.lock '.develop | to_entries[] | .key + .value.version' | grep isort)" ]]; then \
+		$(MAKE) python/isort/check/pipenv; \
+	else \
+		$(MAKE) python/isort/check/docker; \
+	fi;
+
+.PHONY: python/isort/check/pipenv
+python/isort/check/pipenv:
+	$(PIPENV_RUN) isort $(ISORT_FLAGS) --check-only $(PYTHON_LINT_TARGETS) --diff
+
+.PHONY: python/isort/check/docker
+python/isort/check/docker:
+	@$(DOCKER) pull alphachai/isort
 	@$(DOCKER) run -i --rm --user $$(id -u):$$(id -g) --mount type=bind,src=$$(pwd),dst=/app alphachai/isort $(ISORT_FLAGS) --check-only $(PYTHON_LINT_TARGETS) --diff
 
 .PHONY: python/isort
 ## Fix python imports using isort
 python/isort:
+	@if [[ -n "$$(jq -r <Pipfile.lock '.develop | to_entries[] | .key + .value.version' | grep isort)" ]]; then \
```

Comment:
jq needs to pipe errors to dev/null or check if jq is set
otherwise this is going to break for everybody